docs/guides/cloud.rst

```diff
@@ -77,32 +77,59 @@ If you'd prefer, you can also manage your account via `the EdgeDB Cloud
 web-based GUI <https://cloud.edgedb.com/>`_.
 
 The first time you access the web UI, you will be prompted to log in. Once you
-log in with your GitHub account, you'll see your instance list.
+log in with your account, you'll be on the Instances tab of the front page
+which shows your instance list. The other two tabs allow you to manage your
+Org Settings, and Billing.
```

Comment:
Quoting in the context where we are referring to the name of the tab and lower-casing in the instances where we aren't.
```suggestion
log in with your account, you'll be on the "Instances" tab of the front page
which shows your instance list. The other two tabs allow you to manage your
organization settings and billing.
```